File: casPratique1/src/main/java/bank/BankAccount.java
package bank;

import lombok.Data;

@Data
public class BankAccount {
    private int id;
    private String firstName;
    private String lastName;
    private Double balance;

    public BankAccount(int id, String firstName, String lastName, Double balance) {
        this.id = id;
        this.firstName = firstName;
        this.lastName = lastName;
        this.balance = balance;
    }
}

File: CasPratique_1/Solution1/EasyBank/BankAccount.cs
using System;

namespace EasyBank
{
    public class BankAccount
    {
        private int _id;
        private String _firstName;
        private String _lastName;
        private Double _balance;

        public BankAccount(int id, string firstName, string lastName, double balance)
        {
            this._id = id;
            this._firstName = firstName;
            this._lastName = lastName;
            this._balance = balance;
        }

        public int Id
        {
            get => _id;
            set => _id = value;
        }

        public string LastName
        {
            get => _lastName;
            set => _lastName = value;
        }

        public string FirstName
        {
            get => _firstName;
            set => _firstName = value;
        }
        
        public double Balance
        {
            get => _balance;
            set => _balance = value;
        }
    }
}